src/main/java/duke/command/AddDeadlineCommand.java

```diff
@@ -0,0 +1,37 @@
+package duke.command;
+
+import duke.exception.InvalidParameterException;
+import duke.task.Deadline;
+
+/**
+ * The <code>AddDeadlineCommand</code> is created when the user enters <code>"deadline"</code>. The todo command add a \
+ * deadline task into the list of tasks in {@link duke.task.TaskList}. The user interface will display the new deadline
+ * task that is added.
+ */
+public class AddDeadlineCommand extends AddCommand {
+
+    /**
+     * Constructs a new add deadline command that will be executed in the <code>run</code> method of
+     * {@link duke.main.Duke} with the specified line as a parameter.
```

Comment:
Perhaps try to leave an empty line after the description for every JavaDoc comment.